lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -166,4 +165,15 @@ public void isConnected_shouldReturnFalseBeforeConnected() throws Exception {
     client.disconnect();
     assertThat(client.isConnected()).isFalse();
   }
+
+  @Test public void isConnected_multipleClients_shouldReturnFalseAfterDisconnected() throws
+      Exception {
+    LostApiClient anotherClient = new LostApiClient.Builder(application).build();
+    anotherClient.connect();
+    client.connect();
+    client.disconnect();
+    assertThat(client.isConnected()).isFalse();
+    anotherClient.disconnect();
```

Comment:
What is the purpose of calling `anotherClient.disconnect()` after the assert? Is this necessary? Should there be another assert after it?